### ISSUE TITLE
Add Glados instance as part of Fluffy deployment

### ISSUE THREAD

**Issue by kdeme** [status-im/infra-nimbus#158]
[Glados](https://github.com/ethereum/glados/) is a network health monitoring tool for the Portal Network.

There is a public instance available already: https://glados.ethdevops.io/ . But this uses the Trin client as node to access the network. 
I occasionally test this locally with our Fluffy client, but it would be nice to have a long running version of this using fluffy.

Glados exists out of 4 parts (glados-monitor, glados-audit, glados-cartographer and glados-web) + a postgresql server + Portal node (fluffy) + access to a Execution layer JSON-RPC API provider (full node/service). 

There exists also a docker-compose to set these up (except for the full node/service): https://github.com/ethereum/glados/blob/master/DOCKER_GUIDE.md
~~But Fluffy support is still waiting in a PR: https://github.com/ethereum/glados/pull/214~~

This is not that urgent and more a nice-to-have. It could perhaps also be combined with setting up a Fluffy bridge (which would also require access to a full node), after we refactor/harden our Fluffy bridge.

**Comment by kdeme** [comment]
It would be good to set this up now, to have a proper public "fluffy view" on the Portal network. I have tested locally latest fluffy + latest glados (through docker-compose) and it all works.

I don't really mind whether it gets set up with the Docker compose setup or not, but here are the docs for the docker deployment: https://github.com/ethereum/glados/blob/master/DOCKER_GUIDE.md

For fluffy to be used in the docker compose, we need to set `GLADOS_PORTAL_CLIENT=fluffy`.

Considering both Fluffy & glados development is not going through some release cycle, I think it would be best if we can do updates manually ourselves.

**Comment by jakubgs** [comment]
You didn't ping anyone in the original issue, so nobody saw it.

**Comment by kdeme** [comment]
> You didn't ping anyone in the original issue, so nobody saw it.

Yeah, I know, no worries, holiday season was starting and it wasn't an urgent task, + the docker-compose was not adapted for fluffy yet.

Anyway, now the time has come where it is getting more important to have this set up, which is why I added comment + cc

**Comment by jakubgs** [comment]
@markoburcul you can see context on Portal Bridge setup in this issue:

- https://github.com/status-im/infra-nimbus/issues/182

**Comment by kdeme** [comment]
Looking into this today. There are actually more issues than I expected.

I have some PRs and issues open:
- https://github.com/ethereum/glados/pull/356
- https://github.com/ethereum/glados/pull/359
- https://github.com/ethereum/glados/issues/357
- https://github.com/ethereum/glados/issues/358

**Comment by jakubgs** [comment]
Any news on this @kdeme ?
